package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "jandapress",
-  "version": "3.8.1-alpha",
+  "version": "3.8.2-alpha",
```

```suggestion
  "version": "3.8.1-alpha",
```
Not necessary, give it back

---

Edited